interpreter/cling/include/cling/Interpreter/DynamicLibraryManager.h

```diff
@@ -17,6 +17,9 @@
 
 #include "llvm/Support/Path.h"
 
+// FIXME: Implement debugging output stream in cling.
+constexpr unsigned DEBUG = 0;
```

Comment:
Any chance that this can be
- removed, or
- moved into some namespace,
- and / or moved into the source?